nautobot/users/views.py

```diff
@@ -187,6 +194,7 @@ def get(self, request):
             {
```

Comment:
I presume both `get()` and `post()` on `ChangePasswordView` should check the result of `get_auth_backend()` and return an error if the user isn't using local authentication?

For `get()` I'm guessing this would replace the LDAP check just above this comment?

Reply:
Ah good spot!